### DISCUSSION TITLE
OPT-211-Tenderer ("Name of the tendering party")

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#878] [Q&A] [answered]
What exactly is the added value of the "Name of the tendering party" field (OPT-211-Tenderer)? The name of the tendering party must be entered under "Organisations" anyway. In my view, this only leads to redundant and irritating entries.

**Comment by bertrand-lorentz** [comment, marked as answer]
A tendering party can have several organisations, when several tenderers submit a tender together.

The field OPT-211-Tenderer was added to allow having a "human-friendly" name for the tendering party (for example the name of the consortium).
When a form requires you to indicate a reference to a tendering party (to indicate which one submitted a specific tender), instead of just having a list of identifiers (TPA-0001, TPA-0002, etc.), the user interface can use this field to help you choose the intended tendering party.
This is why OPT-211-Tenderer is indicated as the "caption field" for the "Tendering Party" node.

Please also note that OPT-211-Tenderer is never mandatory, so you are not required to fill it in.

**Reply by JanEmde** [reply]
Ok. Thanks :)
